images/wkdev_sdk/Containerfile

```diff
@@ -80,6 +80,17 @@ RUN sed -i 's/^Types: deb$/Types: deb deb-src/' /etc/apt/sources.list.d/ubuntu.s
     rm -rf WebKit && \
     ${APT_AUTOREMOVE}
 
+# Add Rust environment.
+ENV RUSTUP_HOME="/opt/rust" \
+    CARGO_HOME="/opt/rust" \
+    PATH="/root/.cargo/bin:${PATH}"
```

Comment:
Shouldn't that be `/opt/rust/.cargo/bin` ?

Reply:
I think this actually not needed, because later `/root/.cargo` is removed.

Reply:
I'm going to try not to remove '.cargo' and update the path from '/root/.cargo' to '/opt/rust/.cargo'.